plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_multi_pt_mor.md

```diff
@@ -0,0 +1,60 @@
+## Create script
+Revision: 6f65998117a2d1228fc96d36053bd0d394499afe
+
+```scala
+test("Create MOR table with multiple partition fields with multiple types") {
+  withTempDir { tmp =>
+    val tableName = "hudi_multi_pt_mor"
+    // Save current session timezone and set to UTC for consistency in test
+    val originalTimeZone = spark.conf.get("spark.sql.session.timeZone")
+    spark.conf.set("spark.sql.session.timeZone", "UTC")
+
+    spark.sql(
+      s"""
+         |CREATE TABLE $tableName (
+         |  id INT,
+         |  name STRING,
+         |  price DOUBLE,
+         |  ts LONG,
+         |  -- Partition Fields --
+         |  category STRING,            -- Partition field 1: STRING
+         |  year INT,                   -- Partition field 2: INT
+         |  event_date DATE,            -- Partition field 3: DATE
+         |  batch_id BIGINT,            -- Partition field 4: BIGINT
+         |  metric_value DECIMAL(10,2), -- Partition field 5: DECIMAL
+         |  event_timestamp TIMESTAMP,  -- Partition field 6: TIMESTAMP
+         |  is_feature_enabled BOOLEAN  -- Partition field 7: BOOLEAN
+         |) USING hudi
+         | LOCATION '${tmp.getCanonicalPath}'
+         | TBLPROPERTIES (
+         |  primaryKey = 'id,name',
+         |  type = 'mor',
+         |  preCombineField = 'ts'
+         | )
+         | PARTITIONED BY (category, year, event_date, batch_id, metric_value, event_timestamp, is_feature_enabled)
+     """.stripMargin)
+
+    // Configure Hudi properties
+    spark.sql(s"SET hoodie.parquet.small.file.limit=0") // Write to a new parquet file for each commit
+    spark.sql(s"SET hoodie.metadata.compact.max.delta.commits=1")
+    spark.sql(s"SET hoodie.metadata.enable=true")
+    spark.sql(s"SET hoodie.metadata.index.column.stats.enable=true")
+    spark.sql(s"SET hoodie.compact.inline.max.delta.commits=9999") // Disable compaction plan trigger
+
+    // Update column list for column stats index
+    spark.sql(s"SET hoodie.metadata.index.column.stats.column.list=_hoodie_commit_time,_hoodie_partition_path,_hoodie_record_key,id,name,price,ts,category,year,event_date,batch_id,metric_value,event_timestamp,is_feature_enabled")
```

Comment:
What's the reason of setting this explicitly?

Reply:
IIRC, col stats only uses first few columns. Setting this so that all columns will get a col stats index.

Reply:
Removed this as we are not testing column stats index using this table

Reply:
Got it